kmelia/kmelia-war/src/main/java/org/silverpeas/components/kmelia/control/KmeliaSessionController.java

```diff
@@ -1069,40 +1066,47 @@ private synchronized void orderPubs(int sortType) {
     List<KmeliaPublication> publicationsToSort = new ArrayList<>(publications);
 
     int sort = sortType;
-    if (isManualSortingUsed(publicationsToSort) && sort == -1) {
+    boolean manualSortingUsed = isManualSortingUsed(publicationsToSort);
+
+    if (manualSortingUsed && !getSort().isExplicitSort()) {
       // display publications according to manual order defined by admin
       sort = 99;
```

Comment:
It would be nice to use here the Sort.SORT_MANUAL constant instead of an explicit value